docs/core/preview3/tools/dotnet-pack.md

```diff
@@ -75,6 +69,10 @@ Configuration to use when building the project. If not specified, will default t
 
 Updates the star in `-*` package version suffix with a specified string.
 
+`--verbosity <LEVEL>`
+
+Set the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
```

Comment:
It was a cut-and-paste from the CLI repo ...

https://github.com/dotnet/cli/blob/cc80822c16b955a1f664f8a65ed2c76c3c52a824/src/dotnet/commands/dotnet-msbuild/LocalizableStrings.cs#L8

I'll open an issue with them tomorrow.